### PR TITLE
Organized README

### PR DESCRIPTION
**Pull request by avinashdesireddy**
Play-with-docker reference is added and organized the references.

### CODE REVIEW

**Comment by avinashdesireddy** [comment]
#2

File: README.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 # Link Extractor
 
-This repository is a [Docker](https://www.docker.com/) tutorial that illustrates some basics of Docker application containerization platform.
+Link Extractor app scrapes links from a given web page. This repository illistrates step by step approach to learn [Docker](https://www.docker.com/).
```
Comment (ibnesayeed) [inline README.md]: Split sentences in separate lines (not mandatory, but is the undocumented chosen style in this repo). Also, fixing a typo.

```suggestion
Link Extractor app scrapes links from a given web page.
This repository illustrates step by step approach to learn [Docker](https://www.docker.com/).
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+
 It starts from running a very basic script and gradually leads to a multi-service container orchestration (AKA micro-service architecture).
-Follow instructions in the `README` files of various folders from `step0` to `step6` and explore corresponding code.
+
+## How to Use the repository
```
Comment (ibnesayeed) [inline README.md]: Correct letter cases in the heading and add a blank line after it.

```suggestion
## How to Use the Repository

```

File: README.md [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
+1. [Play with Docker Classroom](https://training.play-with-docker.com/microservice-orchestration/) helps to start your docker journey without any additional overhead of configuring docker locally.
+2. Follow instructions in the `README` files of various folders from `step0` to `step6` and explore corresponding code.
```
Comment (ibnesayeed) [inline README.md]: The two options are mutually exclusive, so I would start the second point a little differently.

```suggestion
2. Alternatively, if running on a local Docker installation, follow instructions in the `README` files of various folders from `step0` to `step6` and explore corresponding code.
```

File: README.md [hunk 4]
```diff
@@ -1,9 +1,14 @@
 # Link Extractor
 
-This repository is a [Docker](https://www.docker.com/) tutorial that illustrates some basics of Docker application containerization platform.
+Link Extractor app scrapes links from a given web page. This repository illistrates step by step approach to learn [Docker](https://www.docker.com/).
+
 It starts from running a very basic script and gradually leads to a multi-service container orchestration (AKA micro-service architecture).
-Follow instructions in the `README` files of various folders from `step0` to `step6` and explore corresponding code.
+
+## How to Use the repository
+1. [Play with Docker Classroom](https://training.play-with-docker.com/microservice-orchestration/) helps to start your docker journey without any additional overhead of configuring docker locally.
```
Comment (ibnesayeed) [inline README.md]: Proper casing of Docker as a proper noun.

```suggestion
1. [Play with Docker Classroom](https://training.play-with-docker.com/microservice-orchestration/) helps to start your Docker exploration journey without any additional overhead of installing and configuring Docker locally.
```

File: README.md [hunk 5]
```diff
@@ -1,9 +1,14 @@
...
+2. Follow instructions in the `README` files of various folders from `step0` to `step6` and explore corresponding code.
 To run illustrated codes `Docker` and `Compose` need to be installed on the machine.
 
+## References
```
Comment (ibnesayeed) [inline README.md:14]: A blank line after the heading.

```suggestion
## References

```

**Review by ibnesayeed — CHANGES_REQUESTED** [review]
Thanks for this PR. Please accept my suggestions or make them yourself in a follow up commit to the PR.

**Comment by ibnesayeed** [comment]
Read about incorporating suggested changes at https://help.github.com/articles/incorporating-feedback-in-your-pull-request/

**Comment by avinashdesireddy** [comment]
@ibnesayeed applied the suggestions. Please review and merge the PR.

**Review by ibnesayeed — APPROVED** [review]

**Comment by ibnesayeed** [comment]
LGTM! Thanks @avinashdesireddy for the PR.

**Pull request merged**